CHANGELOG.md

```diff
@@ -21,12 +21,16 @@
   - [#181](https://github.com/Datatamer/unify-client-python/issues/181) Support for seeing a dataset's usage
   - [#202](https://github.com/Datatamer/unify-client-python/issues/202) Support for refreshing published cluster stats
   - [#194](https://github.com/Datatamer/unify-client-python/issues/194) Allowing additional JSON parameters to be used for update of records
+
+  - [#111] (https://github.com/Datatamer/unify-client-python/issues/111) Support for schema mapping unified attributes
```

Comment:
The space here between ] and ( means the markdown won't render properly (also there are extra newlines)